packages/openapi-spec/src/openapi-spec-v2.ts

```diff
@@ -7,16 +7,43 @@
  * OpenApiSpec - A typescript representation of OpenApi/Swagger Spec 2.0
  */
 
-// NOTE(bajtos) Custom extensions can use arbitrary type as the value,
-// e.g. a string, an object or an array
+// tslint:disable:max-line-length
+
+/**
+ * Custom extensions can use arbitrary type as the value,
+ * e.g. a string, an object or an array.
+ */
 // tslint:disable-next-line:no-any
 export type ExtensionValue = any;
 
-
-// tslint:disable-next-line:max-line-length
-// https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md#swagger-object
+/**
+ * This is the root document object for the API specification.
+ * <p>Specification:
```

Comment:
If you are wondering about the `<p>` element: it's a workaround for strong-docs issue https://github.com/strongloop/strong-docs/issues/80